src/status_im/navigation/events.cljs

```diff
@@ -171,3 +171,30 @@
   (let [view-id (if (= view-id :shell-stack) (shell.utils/calculate-view-id) view-id)]
     {:db             (assoc db :view-id view-id)
      :set-view-id-fx view-id}))
+
+(defn- wizard-find-next-screen
+  [db flow-config current-screen]
+  (->> flow-config
+       (filter (fn [{:keys [skip-step? screen-id]}]
+                 (and (not= screen-id current-screen)
+                      (not (and (fn? skip-step?) (skip-step? db))))))
+       first))
+
+(rf/reg-event-fx
+ :navigation/wizard-forward
+ (fn [{:keys [db]} [{:keys [current-screen flow-config start-flow?]}]]
+   (let [next-screen (wizard-find-next-screen db flow-config current-screen)]
+     {:fx [[:dispatch
+            (if start-flow?
+              [:open-modal (:screen-id next-screen)]
+              [:navigate-to-within-stack [(:screen-id next-screen) current-screen]])]]})))
```

Comment:
Should it be part of navigation or some other namespace `wizard/utils` etc. It directly don't look navigation but a specific logic :thinking:. @J-Son89 ?

Reply:
Sure it's not core navigation. If you want to move it out of navigation then,
status-im/common/wizard seems like an appropriate place 👍

Reply:
> Sure it's not core navigation. If you want to move it out of navigation then, status-im/common/wizard seems like an appropriate place 👍

cc @mmilad75

Reply:
Done